x-pack/plugins/ml/public/cases/register_anomaly_swim_lane_attachment.tsx

```diff
@@ -0,0 +1,118 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { i18n } from '@kbn/i18n';
+import { FormattedMessage } from '@kbn/i18n-react';
+import { EuiButtonIcon, EuiDescriptionList } from '@elastic/eui';
+import { CasesUiSetup } from '@kbn/cases-plugin/public';
+import { CoreStart } from '@kbn/core/public';
+import { PersistableStateAttachmentViewProps } from '@kbn/cases-plugin/public/client/attachment_framework/types';
+import { FIELD_FORMAT_IDS } from '@kbn/field-formats-plugin/common';
+import { MlStartDependencies } from '../plugin';
+import { ANOMALY_SWIMLANE_EMBEDDABLE_TYPE, AnomalySwimlaneEmbeddableInput } from '..';
+import { PLUGIN_ICON } from '../../common/constants/app';
+import { getAnomalySwimLaneEmbeddableComponent } from '../embeddables/anomaly_swimlane';
+
+const AttachmentActions: React.FC = () => {
+  return (
+    <EuiButtonIcon
+      data-test-subj="test-attachment-action"
+      onClick={() => {}}
+      iconType="boxesHorizontal"
+      aria-label="See attachment"
+    />
+  );
+};
+
+export function registerAnomalySwimLaneCasesAttachment(
+  cases: CasesUiSetup,
+  coreStart: CoreStart,
+  pluginStart: MlStartDependencies
+) {
+  cases.attachmentFramework.registerPersistableState({
+    id: ANOMALY_SWIMLANE_EMBEDDABLE_TYPE,
+    icon: PLUGIN_ICON,
+    // TODO check where this name is presented
```

Comment:
@jonathan-buttner do you know where this is displayed to the user, if anywhere?

Reply:
Hmm, doesn't look like we use it currently. I would have expected us to use it in this file, but I don't see it referenced: x-pack/plugins/cases/public/components/user_actions/comment/registered_attachments.tsx

Reply:
Hey! It is not used at the moment. It is for future usage.